### PR TITLE
Fix opentelemetry traces examples

### PR DESCRIPTION
**Pull request by ashmeet-kandhari**
Commit Message: 
- Fix opentelemetry traces example
Additional Description:
- Currently running `docker compose up` from `examples/opentelemetry` folder the traces don't appear
- With this change the traces would start appearing
Risk Level:
- Low
Testing:
- N/A
Docs Changes:
- N/A
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
#35497
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Review by wbpcode — COMMENTED** [review]
Thanks. LGTM. But seems that your `examples/opentelemetry/docker-compose.yaml` lost the ending empty line

File: examples/opentelemetry/docker-compose.yaml [hunk 1]
```diff
@@ -69,3 +69,6 @@ services:
     command: ["--config=/etc/otel-collector-config.yaml"]
     ports:
     - "${PORT_UI:-55679}:55679"
+    volumes:
+      - "./otel-collector-config.yaml:/etc/otel-collector-config.yaml"
```
Comment (phlax) [inline examples/opentelemetry/docker-compose.yaml:73]: i think this isnt necessary and is not what we want - probably it wont work

for container configs - at least where they run as non-root - the configs are loaded in the Dockerfile recipe and chmodded/owned - the reason for this is that some users have a strict umask set so loading from the filesystem as a volume doesnt work

File: examples/opentelemetry/docker-compose.yaml [hunk 2]
```diff
@@ -69,3 +69,6 @@ services:
...
+
```
Comment (phlax) [inline examples/opentelemetry/docker-compose.yaml:74]: all files in envoy are expected to have a newline - it doesnt have to be an empty line - it really is best to configure your editor as suggested by the ci:

```console
  Please fix your editor to ensure:

    - no trailing whitespace
    - no preceding mixed tabs/spaces
    - all files end with a newline
```

File: examples/opentelemetry/otel-collector-config.yaml [hunk 3]
```diff
@@ -9,7 +9,9 @@ receivers:
   otlp:
     protocols:
       grpc:
+        endpoint: 0.0.0.0:4317
```
Comment (phlax) [inline examples/opentelemetry/otel-collector-config.yaml:12]: are these necessary?

**Review by phlax — COMMENTED** [review]
thanks for working on this @ashmeet-kandhari 

you need to fix DCO (signoff) on your commits

File: examples/opentelemetry/otel-collector-config.yaml (hunk 3 above)
Comment (ashmeet-kandhari) [reply examples/opentelemetry/otel-collector-config.yaml:12]: Yes without these endpoints I was not able to see the traces even after following the documentation

File: examples/opentelemetry/docker-compose.yaml (hunk 1 above)
Comment (ashmeet-kandhari) [reply examples/opentelemetry/docker-compose.yaml:73]: Ah didn't know about it

**Comment by RyanTheOptimist** [comment]
Looks like format is failing:
```
CodeChecker ERROR [yamllint] examples/opentelemetry/docker-compose.yaml (empty-lines 72:1): too many blank lines (1 > 0)
```

/wait

**Comment by phlax** [comment]
@ashmeet-kandhari im in the process of moving the examples -> https://github.com/envoyproxy/examples/

the PR looks good other than the ci fails - do you want to reraise it in the new repo - if not im happy to pick it up

**Comment by ashmeet-kandhari** [comment]
Hi @phlax ,
I can raise a new PR

**Comment by phlax** [comment]
closing as PR is being raised in examples repo

**Pull request closed without merging**